Fits/Interceptors/[ADV-5] Raptor.md

```diff
@@ -0,0 +1,38 @@
+# Raptor
+
+High DPS Interceptor
+
+
+## Description
+
+A very fast Interceptor focued on burning pings and repositioning, with DPS to target/catch/kill enemy EWAR frigates and long range disruption.
+
+## Fit
+
+```
+[Raptor, [ADV-5] Raptor]
+
+Damage Control II
+Vigor Compact Micro Auxiliary Power Core
+Overdrive Injector System II
+
+5MN Quad LiF Restrained Microwarpdrive
+1MN Afterburner II
```

Comment:
IMO afterburner is a waste of a slot, I've also seen a 2X small shield extender fit